sys-apps/ucspi-ssl/ucspi-ssl-0.10.11.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit fixheadtails qmail
```

Comment:
Was fixheadtails.eclass used here somewhere? I couldn't spot it. So if not, you can remove it.